docs/DeveloperGuide.adoc

```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |view my top five most accessed contact |find them more easily
+
+|`* *` |user |see frequently used contacts instantly |locate them more easily
+
+|`* *` |user |view my contact list online |so that I can view it without the device running the software
+
+|`* *` |user |send email to a group of contacts |easily send email to a group
+
+|`* *` |user |include more details for a person's information |identify the person easily
```

Comment:
"...for a person's contact information"
"store more information about a person and identify him/her more easily"